test/rendering/utils/run-rendering-tests.ts

```diff
@@ -67,10 +68,8 @@ const runRenderingTests = (theme: Object | Function, themeType: String, language
         const element = page.locator('text=Zocalo');
         await element.scrollIntoViewIfNeeded();
       }
-      // Playwright captures screenshot
-      const img = await playwright.screenshot();
-      // Compare screenshot with baseline image
-      expect(img).toMatchSnapshot(`${name}.png`);
```

Comment:
toMatchSnapshot() now should be used for comparing text or arbitrary binary data
https://playwright.dev/docs/test-snapshots